core/components/atoms/editable/Editable.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { BaseProps, extractBaseProps } from '@/utils/types';
+
+export interface EditableProps extends BaseProps {
+  editing?: boolean;
+  renderChildren: () => React.ReactNode;
```

Comment:
Since you are not passing any state to the function, remove it.
Use `children` instead.